examples/quantum_annealing/Running_large_problems_using_QBSolv.ipynb

```diff
@@ -168,9 +171,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "conda_braket",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "conda_braket"
+   "name": "python3"
```

Comment:
This is probably an inadvertent change of meta data from your editor but please revert back to conda_braket